doc/getting_started/installation_linux.rst

```diff
@@ -1,7 +1,7 @@
 .. _installation_linux:
 
-Install Linux Host Dependencies
-###############################
+Install the required tools on Linux
```

Comment:
This PR is generally replacing section capitalization with uncapitalized text. Why?

Why is this an improvement also? Are users unfamiliar with the host vs target distinction?